myaws/ssm_parameter.go

```diff
@@ -91,3 +91,23 @@ func (client *Client) getSSMParametersPerChunk(names []*string, withDecryption b
 
 	return response.Parameters, nil
 }
+
+// GetParametersByPath returns an array of parameters parameters at once
```

Comment:
Why does the word `parameters` are duplicated?
In addition, the Godoc style comment should be a complete sentence, that is, it ends with a period.

Reply:
It seems that this comment was copied from GetSSMParameters, but the difference with the GetSSMParameters function is unclear to reader.

Reply:
Thanks. Updated comment.